src/ServiceControlInstaller.Engine/Validation/QueueNameValidator.cs

```diff
@@ -112,17 +118,23 @@ internal void CheckQueueNamesAreUniqueWithinInstance()
         internal void CheckQueueNamesAreNotTakenByAnotherAuditInstance()
         {
             var allQueues = AuditInstances.SelectMany(instance => new QueueNameValidator(instance).queues);
-            CheckQueueNamesAreNotTaken(allQueues);
+            var duplicates = (
+                from queue in queues
+                where allQueues.Any(p =>
+                    p.QueueType != QueueType.Audit &&
+                    string.Equals(p.ConnectionString, queue.ConnectionString, StringComparison.OrdinalIgnoreCase) &&
+                    string.Equals(p.QueueName, queue.QueueName, StringComparison.OrdinalIgnoreCase) &&
+                    string.Compare("!disable", queue.QueueName, StringComparison.OrdinalIgnoreCase) != 0 &&
+                    string.Compare("!disable.log", queue.QueueName, StringComparison.OrdinalIgnoreCase) != 0)
```

Comment:
Why was `!disable.log` introduced. So far we never needed this

Reply:
Don't know :) legacy code?